experimental/dds/tree2/src/feature-libraries/editable-tree-2/lazyField.ts

```diff
@@ -245,6 +245,15 @@ export class LazySequence<TTypes extends AllowedTypes>
 		makePropertyEnumerableOwn(this, "asArray", LazySequence.prototype);
 	}
 
+	public at(index: number): UnboxNodeUnion<TTypes> | undefined {
+		if (index < -this.length || index >= this.length) {
+			return undefined;
+		}
+		const finalIndex = index < 0 ? index + this.length : index;
+		return inCursorNode(this[cursorSymbol], finalIndex, (cursor) =>
```

Comment:
If you haven't already, you'll want to check that non-integer values are handled compatibly (e.g., 1.5).